projects/frontend/data-pipelines/gui/package.json

```diff
@@ -108,6 +108,7 @@
     "@typescript-eslint/parser": "5.26.0",
     "ajv": "8.11.0",
     "angular-oauth2-oidc": "13.0.1",
+    "axios": "0.27.2",
```

Comment:
Why was axios added? Don't see any direct usages...

Reply:
it's used in the e2e tests, which you propagate with PR https://github.com/vmware/versatile-data-kit/pull/1705